src/seeds.proposals.cpp

```diff
@@ -547,7 +552,7 @@ void proposals::evalproposal (uint64_t proposal_id, uint64_t prop_cycle) {
             proposal.passed_cycle = prop_cycle;
           }
           proposal.executed = false;
-          proposal.staked = asset(0, seeds_symbol);
+          proposal.staked = asset(pitr->staked.amount - stackToBurn.amount, seeds_symbol);
           proposal.status = status_rejected;
```

Comment:
Ok but it should also send back the stake now. Or does this statement alone allow the user to claim back their stake?

Reply:
I updated the ticket to make this much clearer. Stake should be returned.

Reply:
Thanks i wasn't that sure if it was suppose to store de value of the remained or should be returned, I fix this in the new commit too